packages/inquirer/src/types.mts

```diff
@@ -47,7 +47,7 @@ export interface QuestionMap {
 type KeyValueOrAsyncGetterFunction<T, k extends string, A extends Answers> =
   T extends Record<string, any> ? T[k] | AsyncGetterFunction<T[k], A> : never;
 
-export type AnyQuestion<A extends Answers, Type extends string = string> = {
+export type AnyQuestion<A extends Answers = Answers, Type extends string = string> = {
```

Comment:
Do we want to rename it to match what `@type/inquirer` used?

Reply:
AnyQuestion is been exported as Question.
Answers matches too.

Question has a more advanced type called DistinctQuestion which is the current BuiltInQuestion with name property.
I prefer BuiltInQuestion over DistinctQuestion.

Reply:
DistinctQuestion was used in `@types/inquirer`. I honestly do not care so long as we get the type, but if its all the same, i'd use the same one that was used before for the sake of "easier" transition.